errata_tool/connector.py

```diff
@@ -154,10 +154,10 @@ def _get(self, u, **kwargs):
         return_json_decoded_data = True
 
         if kwargs is not None:
-            if 'data' in kwargs:
+            if 'params' in kwargs:
```

Comment:
This change of data to params is likely to break other users who directly or indirectly call through to _get for functionality.  Also the Recognized kwargs docstring lists what is recognized.  If there is a good reason for params to be picked up and treated same/similar to how data is processed we could add that as an additional recognized kwargs.

Reply:
let me double check this.

Reply:
yea after digging a bit more and reviewing the commit message for why.  I agree that it's not used in shale and likely not used by calling methods but would be cautious and maybe we should give deprecation warning if it is provided to help debug issues.  also the docstring should be reflected if we do change the name.

Reply:
Thanks for looking into this @yazug .

I considered keeping `data`, but it was going to make that code even more complicated and difficult to unit-test. Incidentally this pull request led me to look again at replacing the custom mocks with `requests-mock` (issue #116), because right now several of these `ErrataConnector` methods (like `_get()`) are untested, because we just mock them out.

I stepped through this with Wireshark to see how python-requests handles `data` in `requests.get()`. When I use the `data` keyword argument with `requests.get('...', data={'foo': 'bar'})`, then requests sets `Content-Type: application/x-www-form-urlencoded` header and sends a form body.

```
GET / HTTP/1.1
Host: 127.0.0.1:8000
User-Agent: python-requests/2.25.1
Accept-Encoding: gzip, deflate
Accept: */*
Connection: keep-alive
Content-Length: 7
Content-Type: application/x-www-form-urlencoded

foo=bar
```

Forms are usually sent in POST requests, so this is pretty uncommon. It's unlikely the ET Rails app supports GET requests with bodies, but technically [the HTTP specification doesn't disallow sending bodies in GET requests](https://stackoverflow.com/a/983458). In the absence of further information, I will keep `data` in for now. But I will edit this so that users cannot pass `params` with `data` so we keep this code simple, and I'll add that `DeprecationWarning` in a separate pull request.

Reply:
perfect and makes sense to me now.  so +2 from me